lib/pbench/server/api/__init__.py

```diff
@@ -147,39 +147,41 @@ def __init__(self):
 
     def post(self):
         global app
-        graphQL = f"{app.config_graphql.get('host')}:{app.config_graphql.get('port')}"
+        graphQL = (
+            f"http://{app.config_graphql.get('host')}:{app.config_graphql.get('port')}"
+        )
         json_data = request.get_json(silent=True)
 
         if not json_data:
-            app.logger.error("GraphQL: Invalid json object %s", request.url)
+            app.logger.warning("GraphQL: Invalid json object {}", request.url)
```

Comment:
I always think of `warning` and `error` as representing __our__ problem, in which case using them to indicate failures of external servers or user input is misleading. I sometimes wish there was a good additional category for these, but I usually end up going with `info` so it's clear that we saw something "interesting" but it's not our problem to fix. (Though of course it might indicate a problem we can avoid or mitigate or detect more effectively... presuming we include enough context.)

Reply:
I agree with `error` representing _our_ problem, for sure.  We have been using warning to indicate a broad category of problems that include server operations and client behaviors that should be flagged.

Using `WARNING` makes it easier to pull out from all the `INFO`s, but perhaps we should introduce our own "level", `NOTICE`, which is much like the `syslog` `NOTICE` level, were we can reserve `WARNING` and above for our problems, and `NOTICE` could be used for this kind of in between reporting.

If enough of us like the `NOTICE` level, then we'll have to do some additional work to plumb that.  For this commit, I'd like to stick with `WARNING` so that we can get these behavioral changes to land.

Reply:
That's fine; `NOTICE` sounds like a good idea, but I can deal with your scheme in the meantime.